### DISCUSSION TITLE
Need to open as Administrator all of a sudden... What caused this?

### DISCUSSION THREAD

**Discussion by cmarabate** [LukeCroteau/RaidExtractor#65] [Q&A] [answered]
All of a sudden unless I open the RaidExtractor as an Administrator, I get the following error after clicking "Save JSON"...
![image](https://user-images.githubusercontent.com/12236062/117860252-12fa5e00-b25e-11eb-8021-8e30004abdb5.png)

Does anyone know if Plarium Play has upped its security or something which made this necessary? This is my guess because I noticed that my AHK script that I used to replay stages over and over has stopped working and I have a feeling both issues have the same cause.

Or it could just be my PC, has anyone else noticed this?

**Comment by LukeCroteau** [comment, marked as answer]
I'm not seeing this problem right now.
Did your shortcuts change for Raid to open as admin?

**Reply by cmarabate** [reply]
I just tried again and I didn't need to open as admin this time. Not sure what the issue was that day but it apparently fixed itself.

**Comment by cmarabate** [comment]
It has been happening again. Must be something specific to my PC's security settings but I never had this issue until recently.

**Reply by LukeCroteau** [reply]
Both times this seems to have happened *right after* an Update of some kind.
Today happened after a Server Reset.
Maybe something is kicking permissions up when has closed?

**Reply by cmarabate** [reply]
> Both times this seems to have happened right after an Update of some kind.
> Today happened after a Server Reset.

Wait, so I am not the only one!?

> Maybe something is kicking permissions up when has closed?

🤷‍♂️

**Reply by fjjat** [reply]
This program is injecting (Memory) of an other program (RAID). So your User can be Administrator, but Windows doesn't allow direct access to do this. Like a Sandbox. The only way to do something like this, is to run as 'Administrator' -> Machine Account.
Simply Fact 'Admin-User' hasn't the full control over the PC.

**Reply by LukeCroteau** [reply]
> This program is injecting (Memory) of an other program (RAID). So your User can be Administrator, but Windows doesn't allow direct access to do this. Like a Sandbox. The only way to do something like this, is to run as 'Administrator' -> Machine Account. Simply Fact 'Admin-User' hasn't the full control over the PC.

This is 100% incorrect.
Windows allows access to read the memory of other processes. There's an entire ecosystem of apps built to do this, just like Raid Extractor, that do NOT need Administrator access.

Extractor does not use Injection of any type. All it does is read from available memory signatures.

It also does not require administrator access. We've discussed this problem many times, and what I said initially was correct: Raid is being opened in Administrator mode, which is why all of a sudden he needs to run Extractor As Admin.

When Raid is run As Admin, its memory gets protected from non-admin processes.

**Reply by fjjat** [reply]
Sorry, injecting may be the wrong wording OK.
